### PR TITLE
Feat/manual sas payout

### PR DESCRIPTION
**Pull request by falklinus**
# Notion Ticket: [CLX-XXX]

## What?
- Create tool for performing manual SAS payouts to members

## Why?
-

## If applicable, rooms for improvement?
- 

## Optional checklist
- [ ] Codescouted
- [ ] Unit tests written
- [ ] Tested locally

### CODE REVIEW

File: src/portals/hope/pages/tools/ManualSasPayoutPage.tsx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+`
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  width: max-content;
+`
+
+type ManualSASPayout = 'WELCOME' | 'PROMOTION' | 'MONTHLY'
```
Comment (alebedev) [inline src/portals/hope/pages/tools/ManualSasPayoutPage.tsx:43]: would be nice to have "type" in the name. 
 `SasPayoutType`?

File: src/portals/hope/pages/tools/ManualSasPayoutPage.tsx [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+        error: ({ message }) => extractErrorMessage(message),
+        success: 'Payout performed',
+      },
+    )
+  }
+
+  return (
+    <Wrapper>
+      <MainHeadline>Manual SAS payout</MainHeadline>
+      <StyledForm
```
[the remark this answers is not in the document]
Comment (falklinus) [reply src/portals/hope/pages/tools/ManualSasPayoutPage.tsx:78]: I like controlled fields, it's what we do most often. But I'm open to discuss it.

File: src/portals/hope/pages/tools/ManualSasPayoutPage.tsx [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+      >
+        <Input
+          label="Member id"
+          placeholder="Enter member id..."
+          value={memberId}
+          onChange={({ target: { value } }) => setMemberId(value)}
+        />
+        <Flex direction="column">
+          <Label>Type</Label>
+          <Dropdown>
```
Comment (alebedev) [inline src/portals/hope/pages/tools/ManualSasPayoutPage.tsx:92]: Do we have some higher-level select component?  Assigning `onClick` and `selected` to every items seems repetitive

Comment (falklinus) [reply src/portals/hope/pages/tools/ManualSasPayoutPage.tsx:92]: Could make the "options" into an array and map, that's mostly how we use it. There is also a select component I think, but this one is the most similar in style to the <Input />.

File: src/portals/hope/pages/tools/ManualSasPayoutPage.tsx [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+        )}
+
+        <Button disabled={loading} type="submit">
+          Submit
+        </Button>
+      </StyledForm>
+
+      <Spacing top="medium" />
+
+      {!!data?.manualSasBonusPayout && (
```
Comment (alebedev) [inline src/portals/hope/pages/tools/ManualSasPayoutPage.tsx:141]: Nice!  And if there's an error, do we want to show it too?

Comment (falklinus) [reply src/portals/hope/pages/tools/ManualSasPayoutPage.tsx:141]: yeah if the api returns some useful error we can show it, haven't even tested it yet 👍

**Review by alebedev — APPROVED** [review]
Looks good to me.  None of my comments are must-solve, mostly styling suggestions

**Review by lucas-larsson — APPROVED** [review]

**Pull request merged**